modules/nf-core/wittyer/environment.yml

```diff
@@ -0,0 +1,5 @@
+name: wittyer
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
```

Comment:
I removed the dependencies analogous to `bcl2fastq` but that leads to a linting error.

Reply:
I now added `tabix` as a dependency but there should be a better solution.